ocean_provider/utils/test/test_util.py

```diff
@@ -73,15 +65,8 @@ class Dummy:
     filename = "<<filename>>"
     url_object = {"url": f"https://source-lllllll.cccc/{filename}", "type": "url"}
     _, instance = FilesTypeFactory.validate_and_create(url_object)
-    with patch(
-        "ocean_provider.file_types.definitions.is_safe_url",
-        side_effect=[True],
-    ):
-        with patch(
-            "requests.get",
-            side_effect=[mocked_response],
-        ):
-            response = instance.build_download_response(request)
+    with patch("requests.get", side_effect=[mocked_response]):
```

Comment:
same comment here and in other places throughout this file.

Reply:
Please see my response to your comment on line 56 of this file.